package.json

```diff
@@ -33,6 +33,16 @@
     "deduplicate": "yarn-deduplicate --strategy=highest",
     "postinstall": "yarn patch-package"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "yarn.lock": [
+      "yarn yarn-deduplicate --strategy=highest"
```

Comment:
why not just run `yarn deduplicate`, and use our existing script? This seems more prone to skew.

Reply:
did not know this existed, thanks